tests/dash/configs/privatelink_config.py

```diff
@@ -52,13 +61,53 @@
         "local_region_id": LOCAL_REGION_ID
     }
 }
+APPLIANCE_FNIC_CONFIG = {
+    f"DASH_APPLIANCE_TABLE:{APPLIANCE_ID}": {
+        "sip": APPLIANCE_VIP,
+        "vm_vni": VM_VNI,
+        "outbound_direction_lookup": OUTBOUND_DIR_LOOKUP,
+    }
+}
+
+ENI_FNIC_CONFIG = {
```

Comment:
what is the need for this FNIC without trusted_vni config ? can we do away with this and just use the ENI_FNIC_TRUSTED_VNI_CONFIG ?

Reply:
unused config removed